.github/workflows/publish-github-release.yml

```diff
@@ -0,0 +1,92 @@
+name: Publish GitHub Release as newest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 1-5'
```

Comment:
Given the scheduled trigger, if for some reason we have to change a release back to pre-release or draft, any way we can stop this workflow from re-processing the same release again?

Reply:
🤔 We could follow the [_version.yml ](https://github.com/bitwarden/ios/blob/main/.github/workflows/_version.yml#L109-L126) approach. The gist:
1. When this workflow changes a release, upload a json file with state info - e.g., release tag, release title, initial state (drift / pre-release) and changed to state (release)
2. Before changing a release, get the artifact from the last run of this workflow and confirm if we're about to change the same release.

Reply:
@michalchecinski Was working on something unrelated that required downloading artifacts from the last run of a workflow, remembered this may be useful for you too:

`gh run download $(gh run list --workflow=build.yml --event=schedule --limit 1 --json databaseId --jq '.[0].databaseId')`

* Replacing the workflow filename with `$(basename $GITHUB_WORKFLOW)` may work but I didn't test it. 
* Filtering event name would skip `workflow_dispatch` events we may use to test the workflow.

Reply:
I've implemented the whole setup to do this. But when I was testing it (it works as expected) I realized that the release cannot be converted into a draft once released. So this is unnecessary, it seems. I should have checked earlier 😅